### PR TITLE
Add healthcheck on Dockerfile

### PR DESCRIPTION
**Pull request by Yopii**
Hi,

Quick improvement, add HEALTHCHECK on Dockerfile to monitor the presence of the service.

- [x] Please explain your motives to contribute this change: what problem you are trying to fix, what improvement you are trying to make
- [x] If the PR modifies or adds images, please use the `run-tests.sh imagedir` command to verify the image works
- [x] If the PR modifies or adds images, please try to make sure the images run correctly on Linux

### CODE REVIEW

**Comment by Yopii** [comment]
Hello,

Several PRs are present and not even one comment has been made (rejected, accepted, etc.).

We try to contribute to the project in order to help the community but in the end the PR remains unused, abandoned.

On this PR, I suggest adding a "HEALTHCHECK", very simple, an additional line to improve the image produced. 

@janos-ss, @julienlancelot, @benoit-sns could you at least validate, reject or discuss on the current PRs when you go through this repository?

Thank you.

File: 6.7.6-community/Dockerfile [hunk 1]
```diff
@@ -50,3 +50,4 @@ WORKDIR $SONARQUBE_HOME
 COPY run.sh $SONARQUBE_HOME/bin/
 USER sonarqube
 ENTRYPOINT ["./bin/run.sh"]
+HEALTHCHECK --start-period=30s --interval=2m --timeout=3s --retries=3 CMD curl -f http://localhost:9000/ || exit 1
```
Comment (simonbrandhof) [inline 6.7.6-community/Dockerfile]: The web service `api/system/ping` could be used instead of loading the homepage. It checks that the web server is up.

**Comment by pierre-guillot-sonarsource** [comment]
Thanks for your PR. We plan to improve SQ docker images with the release of DE and EE images. I will add this suggestion to the internal discussion. I'm closing this PR because we wont apply changes to the SQ version of this PR (6.7 and 7.6) but rather on the 7.9 LTS or 8.X version.

I tracked your idea here : https://jira.sonarsource.com/browse/SONAR-12278

Thanks for your contribution :)

**Comment by pierre-guillot-sonarsource** [comment]
https://github.com/docker-library/healthcheck/pull/20

**Pull request closed without merging**